pixeltable/functions/video.py

```diff
@@ -56,7 +56,7 @@ class make_video(pxt.Aggregator):
     def __init__(self, fps: int = 25):
         """follows https://pyav.org/docs/develop/cookbook/numpy.html#generating-video"""
         self.container: Optional[av.container.OutputContainer] = None
-        self.stream: Optional[av.stream.Stream] = None
+        self.stream: Optional[av.video.stream.VideoStream] = None
```

Comment:
While you're here, add class-level member declarations?

Reply:
Sure, I guess. We should probably just do a sweep to go through the code base and add these, if we care about them for code quality.

I tend to resist scope creep for PRs ("as long as we're solving X, let's fix unrelated problem Y") but I went ahead and did this.